raiden/tests/unit/transfer/test_node.py

```diff
@@ -56,3 +114,331 @@ def test_is_transaction_effect_satisfied(
+    state_change = ActionNewTokenNetwork(
+        payment_network_address=payment_network_address, token_network=token_network
+    )
+    transition_result = handle_new_token_network(
+        chain_state=chain_state, state_change=state_change
+    )
+    new_chain_state = transition_result.new_state
+    payment_network = new_chain_state.identifiers_to_paymentnetworks[payment_network_address]
+    assert payment_network.address == payment_network_address
+    assert not transition_result.events
+    assert get_networks(
+        chain_state=chain_state,
+        payment_network_address=payment_network_address,
+        token_address=token_address,
+    ) == (payment_network, token_network)
+
+
+def test_is_transaction_expired():
+    expiration = 24
+    block_number = expiration + 1
+    transaction = ContractSendChannelUpdateTransfer(
+        expiration=expiration,
+        balance_proof=None,
+        triggered_by_block_hash=factories.make_block_hash(),
+    )
+    assert is_transaction_expired(transaction, block_number)
+    transaction = ContractSendSecretReveal(
+        expiration=expiration,
+        secret=factories.UNIT_SECRET,
+        triggered_by_block_hash=factories.make_block_hash(),
+    )
+    assert is_transaction_expired(transaction, block_number)
+
+    transaction = ContractSendSecretReveal(
+        expiration=block_number,
+        secret=factories.UNIT_SECRET,
+        triggered_by_block_hash=factories.make_block_hash(),
+    )
+    assert not is_transaction_expired(transaction, block_number)
+
+
+def test_subdispatch_by_canonical_id(chain_state):
+    our_model, _ = create_model(balance=10, merkletree_width=1)
+    partner_model, _ = create_model(balance=0, merkletree_width=0)
+    channel_state = create_channel_from_models(
+        our_model, partner_model, factories.make_privatekey_bin()
+    )
+    canonical_identifier = channel_state.canonical_identifier
+    token_network = TokenNetworkState(
+        address=canonical_identifier.token_network_address,
+        token_address=factories.make_address(),
+        network_graph=TokenNetworkGraphState(
+            token_network_address=channel_state.token_network_address
+        ),
+    )
+    token_network.partneraddresses_to_channelidentifiers[
+        partner_model.participant_address
+    ] = canonical_identifier.channel_identifier
+    token_network.channelidentifiers_to_channels[
+        canonical_identifier.channel_identifier
+    ] = channel_state
+    payment_network = PaymentNetworkState(
+        address=factories.make_address(), token_network_list=[token_network]
+    )
+    chain_state.identifiers_to_paymentnetworks[payment_network.address] = payment_network
+    chain_state.tokennetworkaddresses_to_paymentnetworkaddresses[
+        canonical_identifier.token_network_address
+    ] = payment_network.address
+    # dispatching a Block will raise an assertion error
+    with pytest.raises(AssertionError):
```

Comment:
As stated somewhere else, with this `token_network` state machine is going to be the only place which raises an assert error here. So i'd rather have it just ignore the change.